src/App.js

```diff
@@ -0,0 +1,93 @@
+import './App.css';
+import List from "./List";
+import Header from "./Header";
+import {useState} from 'react';
+
+let counter = 3;
+
+function App(props) {
+    const data=props.data
+
+    const [toDoItems, setToDoItems] = useState(data);
+    const [completedItemIDs, setCompletedItemIDs] = useState([]);
```

Comment:
Much better to store completed as a boolean field within each task.  You'll need to save it in Firestore, so you'll need to make that change.